homeassistant/components/device_tracker/owntracks.py

```diff
@@ -50,7 +59,7 @@ def validate_payload(payload, data_type):
                           'because of missing or malformatted data: %s',
                           data_type, data)
             return None
-        if data_type == VALIDATE_TRANSITION:
+        if data_type == VALIDATE_TRANSITION or data_type == 'waypoints':
```

Comment:
For compatibility would be nice to use VALIDATE_WAYPOINT - it is used more than once in the code.